packages/backend/src/exploreCompiler.test.ts

```diff
@@ -0,0 +1,50 @@
+import {
+    exploreCircularReference,
+    exploreComplexReference,
+    exploreComplexReferenceCompiled,
+    exploreMissingBaseTable,
+    exploreMissingJoinTable,
+    exploreOneEmptyTable,
+    exploreOneEmptyTableCompiled,
+    exploreReferenceDimension,
+    exploreReferenceDimensionCompiled,
+    exploreReferenceInJoin,
+    exploreReferenceInJoinCompiled,
+    exploreTableSelfReference,
+    exploreTableSelfReferenceCompiled
+} from "./exploreCompiler.mock";
+import {compileExplore} from "./exploreCompiler";
+import {CompileError} from "./errors";
+import {compile} from "morgan";
+
+test('Compile empty table', () => {
```

Comment:
We should define a naming convention for tests. Some examples here:  https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea

I have been using this format or similar:
"Should do X when Y"

Examples:
"Should compile empty table"
"Should throw error when missing base table"